devex/cross-nuget/windows/build.ps1

```diff
@@ -6,9 +6,11 @@
 #    |- Debug
 #    |- Release
 
-$ErrorActionPreference = "Stop"
+Param(
+  [string]$OE_SDK_TAG = 'master'
```

Comment:
How do you override and tell powershell to use a specific release?  The release process requires referencing a released version rather than the non-released current state of the master branch

Reply:
Any suggestions on how I can make it more visible? I could also make it a required argument so it's less error prone. I mentioned it in line 90 of `devex/cross-nuget/README.md`, but I realize now that it's easy to miss/overlook. 

The following command would override the default value with `v0.12.x`:
```
.\build.ps1 -OE_SDK_TAG v0.12.x
```